includes/wc-order-functions.php

```diff
@@ -715,9 +715,13 @@ function wc_restock_refunded_items( $order, $refunded_line_items ) {
 		$item_stock_reduced = $item_stock_reduced - $qty_to_refund;
 
 		if ( 0 < $item_stock_reduced ) {
+			// Keeps track of total running tally of reduced stock.
 			$item->update_meta_data( '_reduced_stock', $item_stock_reduced );
+			// Keeps track of only refunded items that needs restock.
+			$item->update_meta_data( '_restock_refunded_items', $qty_to_refund );
```

Comment:
Shouldn't this be an addition op instead of an update? So if we are tracking the refunded quantity we are supposed to stock back, we would add to the already existing quantity right, instead of updating it? I think in the current code there would be a bug when we :
1. Refund with restock
2. Refund with restock again
3. Update order

It will only keep track of the last restock operation, and will reduce the stock again from our first refund.